jekyll/_cci2/release/releases-overview.adoc

```diff
@@ -246,6 +246,63 @@ The following table shows a complete list of labels and annotations either requi
 |No. Only set if you want to disable the cancel release feature for your component.
 |===
 
+== Autodetected releases and release markers
+
+CircleCI automatically generates release markers for your deployment pipelines. Release markers provide a lightweight way to log your releases without requiring a full CircleCI releases setup. Release markers create for you a log of all releases in one place, for a clear overview of what has changed, without the need to search through your CI/CD pipelines. Release markers log all new releases in one place and link back to the CI/CD pipelines that caused them. You can use release markers independently, without installing the release agent.
+
+=== Autodetection of releases
+
+CircleCI looks for the word `deploy` in your job names to detect when a release may be in progress and creates an autodetected release marker.
```

Comment:
```suggestion
CircleCI looks for the word `deploy` in your job names to detect when a release may have happened and creates an autodetected release marker.
```

clarify the release might have been done for two reasons

1. we decide to create the release when the deploy job is success, it means is in a terminal status (the thing already happened)
2. we don't care much about the status of the release, we just log the event but the release may be in progress, success or failure status.